### DISCUSSION TITLE
calling CQ pota,sota,jota,vota etc?

### DISCUSSION THREAD

**Discussion by QDX2545** [N0BOY/FT8CN#84] [Ideas]
Just installed this and using on a Samsung TabA. Is there a way to call "CQ POTA callsign gridsquare"?

 I cannot find how to do this.

**Comment by N0BOY** [comment]
You can add "POTA" as the CQ Modifier.

![image](https://github.com/N0BOY/FT8CN/assets/115208948/69f2eb92-bd17-4cfc-a417-700b1236a337)

**Comment by QDX2545** [comment]
Thanks, just confirmed it and that makes a lot of sense.
73
Mike
